tools/accuracy_checker/configs/resnet-18.yml

```diff
@@ -0,0 +1,69 @@
+models:
+  - name: resnet-18
```

Comment:
if 'v1' is significant suffix, I'd prefer to have that consistent in model name and model folder. Do you have any objections agains name like resnet-18-v1?